test/e2e/workflow/workflow.test.ts

```diff
@@ -0,0 +1,364 @@
+  });
+
+  afterEach(async () => {
+    await workflowRuntime.stop();
+    await workflowClient.stop();
+  });
+
+  it("should be able to run an empty orchestration", async () => {
+    let invoked = false;
+    const emptyWorkflow: TWorkflow = async (_: WorkflowContext, __: any) => {
+      invoked = true;
+    };
+    workflowRuntime.registerWorkflow(emptyWorkflow);
+    await workflowRuntime.start();
+
+    const id = await workflowClient.scheduleNewWorkflow(emptyWorkflow);
+    const state = await workflowClient.waitForWorkflowCompletion(id, undefined, 30);
+
+    expect(invoked).toBe(true);
+    expect(state).toBeDefined();
+    expect(state?.name).toEqual(getFunctionName(emptyWorkflow));
+    expect(state?.instanceId).toEqual(id);
+    expect(state?.workflowFailureDetails).toBeUndefined();
+    expect(state?.runtimeStatus).toEqual(WorkflowRuntimeStatus.COMPLETED);
+  }, 31000);
+
+  it("should be able to run an activity sequence", async () => {
+    const plusOne = async (_: WorkflowActivityContext, input: number) => {
+      return input + 1;
+    };
+
+    const sequenceWorkflow: TWorkflow = async function* (ctx: WorkflowContext, startVal: number): any {
+      const numbers = [startVal];
+      let current = startVal;
+
+      for (let i = 0; i < 10; i++) {
+        current = yield ctx.callActivity(plusOne, current);
+        numbers.push(current);
+      }
+
+      return numbers;
+    };
+
+    workflowRuntime.registerWorkflow(sequenceWorkflow).registerActivity(plusOne);
+    await workflowRuntime.start();
+
+    const id = await workflowClient.scheduleNewWorkflow(sequenceWorkflow, 1);
+    const state = await workflowClient.waitForWorkflowCompletion(id, undefined, 30);
+
+    expect(state).toBeDefined();
+    expect(state?.name).toEqual(getFunctionName(sequenceWorkflow));
+    expect(state?.instanceId).toEqual(id);
+    expect(state?.workflowFailureDetails).toBeUndefined();
+    expect(state?.runtimeStatus).toEqual(WorkflowRuntimeStatus.COMPLETED);
+    expect(state?.serializedInput).toEqual(JSON.stringify(1));
+    expect(state?.serializedOutput).toEqual(JSON.stringify([1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11]));
+  }, 31000);
+
+  it("should be able to run fan-out/fan-in", async () => {
+    let activityCounter = 0;
+
+    const incrementActivity = (_: WorkflowActivityContext) => {
+      activityCounter++;
+    };
+
+    const sequenceWorkflow: TWorkflow = async function* (ctx: WorkflowContext, count: number): any {
+      // Fan out to multiple sub-orchestrations
+      const tasks: Task<any>[] = [];
+
+      for (let i = 0; i < count; i++) {
+        tasks.push(ctx.callActivity(incrementActivity));
+      }
+
+      // Wait for all the sub-orchestrations to complete
+      yield ctx.whenAll(tasks);
+    };
+
+    workflowRuntime.registerWorkflow(sequenceWorkflow).registerActivity(incrementActivity);
+    await workflowRuntime.start();
+
+    const id = await workflowClient.scheduleNewWorkflow(sequenceWorkflow, 10);
```

Comment:
Please consider using this everywhere (examples, e2e) for readability.

```suggestion
    const id = await workflowClient.scheduleNewWorkflow(sequenceWorkflow, input=10);
```

Reply:
I don't think typescript allows calling method with the arguments passed by name, when I update to above it gives error as 
<img width="1464" alt="image" src="https://github.com/dapr/js-sdk/assets/89094811/4975c8dd-1b3c-45b4-8c38-69bbbc3383d8">

Reply:
Oh yeah, sorry! We could use objects and destructuring to get around this, but I guess this is fine for now. Thanks!